### PR TITLE
Fix print style issues

### PR DESCRIPTION
**Pull request by weeman1337**
* Resolves: #752 
* Target version: master 

### Summary

Fixes several print style issues:
- hides some more elements
- all rows are being printed

* [before.pdf](https://github.com/nextcloud/deck/files/7110145/before.pdf)
* [after.pdf](https://github.com/nextcloud/deck/files/7110146/after.pdf)

### Checklist

- [x] Code is properly formatted
- [x] Sign-off message is added to all commits
- [x] Tests (unit, integration, api and/or acceptance) are included
- [x] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by stefan-niedermann** [comment]
@weeman1337 maybe you can add some before/after screenshots or PDFs for easier review? 🙂 🚀

**Comment by stefan-niedermann** [comment]
Oh i am so sorry, i didn't notice that you already included them!

**Review by stefan-niedermann — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 1.6.0 - unreleased
+
+### Fixed
+* [#752](https://github.com/nextcloud/deck/issues/752) Print Style Error
```
Comment (juliushaertl) [inline CHANGELOG.md:7]: Could you remove the changelog change? The file will be updated during the release process automatically from the changes that then got merged for the release :)

Comment (weeman1337) [reply CHANGELOG.md]: Ah - didn't know that. Done

**Review by juliushaertl — APPROVED** [review]

**Comment by juliushaertl** [comment]
Good one, thanks a lot @weeman1337 and nice to see you around again :wink:

**Pull request merged**